### PR TITLE
Change the design and some logic of the filter panel for issues

### PR DESCRIPTION
**Pull request by VitaliiMaksymiv**
### Before:
image

### After:
image

### CODE REVIEW

**Review by andriyshevchenko — COMMENTED** [review]
Please make typesript code appear each statement on new line in the HTML
зображення

File: src/app/modules/shared/shared.module.ts [hunk 1]
```diff
@@ -122,7 +130,8 @@ import { BreadcrumbComponent } from './components/breadcrumb/breadcrumb.componen
     UserService,
     VehicleTypeService,
     VehicleService,
-    StatisticsService
+    StatisticsService,
+    {provide: MAT_DATE_LOCALE, useValue: 'en-GB'}
```
Comment (igolovchuk) [inline src/app/modules/shared/shared.module.ts]: why en-GB not uk or uk-UA?

**Review by BohdanBuhrii — CHANGES_REQUESTED** [review]
These changes make the element less functional, which is why it does not meet the requirements. Also the quality of the code and its comprehensibility is much worse than before.

File: src/app/modules/shared/components/filters-tabs/filters-tabs.component.html [hunk 2]
```diff
@@ -1,87 +1,144 @@
...
+
+    <div class="column">
+
+      <mat-form-field class="selection">
+        <mat-select placeholder="Група несправності" [(value)]="selectedGroup"
+          (selectionChange)="selectMalfunctionGroupType($event.value);">
+          <mat-option *ngFor="let group of malfunctionGroupList" [value]="group">{{group.name}}</mat-option>
+        </mat-select>
+        <button mat-button *ngIf="selectedGroup" matSuffix mat-icon-button aria-label="Clear"
+          (click)="selectedGroup=undefined;
```
Comment (BohdanBuhrii) [inline src/app/modules/shared/components/filters-tabs/filters-tabs.component.html:90]: move logic to function

**Review by BohdanBuhrii — COMMENTED** [review]
see comments

**Review by BohdanBuhrii — APPROVED** [review]

**Review by igolovchuk — APPROVED** [review]

**Review by andriyshevchenko — APPROVED** [review]

**Pull request merged**